package.xml

```diff
@@ -17,4 +17,5 @@
   <depend>tf</depend>
   <depend>shape_msgs</depend>
 
+  <test_depend>gtest</test_depend>
```

Comment:
http://wiki.ros.org/gtest#Building_and_running_tests

They specifically specify not to declare a dependency on gtest. But does CI still work without this?

Reply:
Yep, CI will still work. Remove it to follow the rules.